commands/ci/index.js

```diff
@@ -26,7 +26,11 @@ const cmd = {
       char: 'w',
       hasValue: false,
       description: 'keep running and watch for new and update tests'
-
+    },
+    {
+      name: 'json',
```

Comment:
Do you mind adding `char: 'j',` so we're consistent with [other `--json` command line flags]( https://github.com/heroku/heroku-apps/blob/master/src/commands/apps/info.js#L136)? Otherwise this looks solid.